modules/oc-mirror-installing-plugin.adoc

```diff
@@ -7,11 +7,11 @@
 [id="installation-oc-mirror-installing-plugin_{context}"]
 = Installing the oc-mirror OpenShift CLI plugin
 
-To use the oc-mirror OpenShift CLI plugin to mirror registry images, you must install the plugin. If you are mirroring image sets in a fully disconnected environment, ensure that you install the oc-mirror plugin on the host with internet access and the host in the disconnected environment with access to the mirror registry.
-
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
```

Comment:
@snarayan-redhat - could you add something like this (please change the wording as you see fit)  under .Prerequesites
"Ensure that the default umask of 0022 is set on the operating system oc-mirror will be executed on"

Reply:
Subhashini is already assigned jira https://issues.redhat.com/browse/OCPBUGS-32411

Reply:
Also could we add something like this in the .Prerequisite

"Ensure you use the correct version rhel8 binary for rhel8 systems and rhel9 for rhel9 operating systems, this is important as rhel9 is FIPS compliant and has differences in static and dynamic library linking "

Reply:
sure, I will ask Subhashini to add both the prereqs under the same bug.

Reply:
Thank you